framework/math/unknown_manager/unknown_manager.cc

```diff
@@ -107,23 +101,21 @@ UnknownManager::SetUnknownNumOffBlockConnections(unsigned int unknown_id, int nu
 }
 
 void
-UnknownManager::SetUnknownComponentNumOffBlockConnections(unsigned int unknown_id,
+UnknownManager::SetUnknownComponentNumOffBlockConnections(int unknown_id,
                                                           unsigned int component,
                                                           int num_conn)
 {
-  auto& log = Logger::GetInstance();
-
   if (unknown_id < 0 or unknown_id >= unknowns.size())
   {
-    log.LogAllError() << "UnknownManager failed call to SetUnknownComponentTextName,"
+    log.LogAllError() << "UnknownManager failed call to SetUnknownComponentName,"
```

Comment:
Can you add a patch that is fixing the member function name to `SetUnknownComponentNumOffBlockConnections` which would be the correct name? Also the same problem on line `118`.

Reply:
Done.